src/core/config.js

```diff
@@ -90,15 +90,6 @@ if ( urlParams.seed ) {
 	}
 }
 
-function getIds ( params ) {
-	var paramArray = [];
-	if ( params ) {
-
-		// Ensure that params is an array
-		paramArray = decodeURIComponent( params ).split( "," );
-	}
-	return paramArray;
-}
-
-config.testId = getIds( urlParams.testId );
-config.moduleId = getIds( urlParams.moduleId );
+// Force array-valued testId and moduleId configuration values
+config.testId = [].concat( urlParams.testId || [] );
+config.moduleId = [].concat( urlParams.moduleId || [] );
```

Comment:
The `urlParams` properties are not guaranteed to be arrays. I could replace the `|| []` with an `if ( urlParams.moduleId )`, but I prefer the single-line form and am not worried about creating a couple of unnecessary arrays during initialization.